oh_staff_ui/templates/oh_staff_ui/item_tree.html

```diff
@@ -0,0 +1,50 @@
+{% for parent, children in relatives.items %}
+    {# if this is topmost parent or immediate parent of page we're on, open the details #}
+    {% if item in children or parent == top_parent %}
+        {% if children %}
+            <li>
+                <details open> 
+                    {# if this is the page we're on, don't link #}
+                    {% if parent == item %}
+                    <summary>{{parent}}</summary>
+                    {% else %}
+                    <summary><a href="{% url 'edit_item' parent.id %}">{{parent}} </a></summary>
+                    {% endif %}
+                    <ul>
+                        {% include "oh_staff_ui/item_tree.html" with relatives=children %}
+                    </ul>
+                </details>
+            </li>
+        {% else %}
+            {# if this is the page we're on, don't link #}
+            {% if parent == item %}
+                <li>{{parent}}</li>
+            {% else %}
+                <li><a href="{% url 'edit_item' parent.id %}">{{parent}}</a></li>
+            {% endif %}
+        {% endif %}
+    {% else %}
+        {% if children %}
```

Comment:
Lines 27-48 are exactly the same as lines 4-25, except the ` open` on line 6 vs line 29.  I think this could just be one block of code, with the test on line 3 moved to what's currently line 6, something like
```
<details {% if item in children or parent == top_parent %} open {% endif %}>
```